### DISCUSSION TITLE
Transferring Stewardship to Astral

### DISCUSSION THREAD

**Discussion by indygreg** [indygreg/python-build-standalone#396] [General]
As explained at https://gregoryszorc.com/blog/2024/12/03/transferring-python-build-standalone-stewardship-to-astral/ I will be transferring this GitHub repository to the @astral-sh organization on 2024-12-17.

Existing links and download URLs should work as before and should be transparent to everyone.

If you have any concerns about this transfer, please speak up here or send me a private email at [gregory.szorc@gmail.com](mailto:gregory.szorc@gmail.com).

**Comment by zanieb** [comment]
Thank you Greg!

We (the Astral team) are very happy to answer any questions or concerns as well. We're passionate about this project and hope that the transfer will help build a stronger community around it.

**Comment by bulletmark** [comment]
To clarify, so will this repo will be archived and newer releases available from `github.com/astral-sh/python-build-standalone`? Will older releases be available there also, or have to be fetched from the archive here?

**Reply by zanieb** [reply]
The plan is to [_transfer_](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository) the repository, not archive it.

Older releases will be available in the new location. Per the linked GitHub documentation:

> All links to the previous repository location are automatically redirected to the new location
